src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,105 @@
-import React from 'react';
 import './NewCommentForm.scss';
```

Comment:
Consider to use [`styled-components`](https://styled-components.com/) or [`css modules`](https://github.com/css-modules/css-modules)
1. Directly importing css file is not a good practice fro component-level styling 
2. Styles are not scoped to the component (global scoped)
3. Harder to achieve component-level styling.